spring-cloud-skipper-server/src/main/java/org/springframework/cloud/skipper/server/app/SkipperServerApplication.java

```diff
@@ -40,6 +43,7 @@
 		SessionAutoConfiguration.class
 	})
 @EnableSkipperServer
+@Import({KubernetesPlatformAutoConfiguration.class, CloudFoundryPlatformAutoConfiguration.class})
```

Comment:
Is this required given these auto configuration classes are now part `spring.factories` from their respective jars?